src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx

```diff
@@ -1,12 +1,16 @@
 ---
-title: Origins
+title: Azion CLI origin
 description: Create, list, describe, update, and delete your origins on the Azion platform through Azion CLI.
 meta_tags: cli, origins, devops
 namespace: documentation_cli_origins
 menu_namespace: cliMenuAlpha
 permalink: /documentation/products/cli/origins-crud/
 ---
 
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case any required field isn't informed through the specific flags, an interactive message prompts and asks for the missing information.
```